docs/wallet_browser_extension/intro.md

```diff
@@ -0,0 +1,52 @@
+# ParallelChain Wallet Extension
+
+ParallelChain Wallet Extension is an extension for managing your XPLL and integrating your DApp.
+It allows your DApp to interact with your DApp users' XPLL accounts and make smart contract calls through the extension.
+
+## Features
+
+- Browser extension
+  - Sending transactions
+  - Staking
+  - Triggering confirmation for smart contract or send token calls
+- [ParallelChain wallet provider API](./provider/intro.md)
+
+## Supported Platforms
+
+- Chrome Browser (Minimum version: Chrome 57)
+- Firefox Browser (Minimum version: Firefox 57.0)
+
+## Getting Started
+
+### Prerequisites
+
+- Chrome or Firefox browser installed (Refer to above for supported versions)
+
+### Extension Installation
+
+#### Chrome
+
+1. Chrome users can only install an approved extension via Chrome Web Store.
+   For the time being, download the `parallelchain_wallet_extension-0.0.1-chromium.zip` file.
```

Comment:
"For the time being" <-- feels a bit unnecessary here, same for the firefox section. 

Also, need to tell users where to find the zip file. (Remember to add this in after the extension is released.)

Reply:
solved by bb00eef